src/pages/ItemDetailPage.jsx

```diff
@@ -1,8 +1,106 @@
-import { useParams } from 'react-router-dom';
+import { Helmet } from 'react-helmet';
+import { useEffect, useState } from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
+import { getDetailProduct, getProductComments } from '/src/api/api';
+import { StyledButton } from '/src/components/common/Button';
+import DetailProduct from '/src/components/DetailProduct';
+import DetailProductComments from '/src/components/DetailProductComments';
+import { Input } from '/src/components/Input';
+import {
+  FlexWrapper,
+  StyledMain,
+  CommentForm,
+} from '/src/components/common/CommonComponents';
+import iconBack from '/src/assets/ic_back.svg';
 
 function ItemDetailPage() {
-  //   const { productId } = useParams();
-  return <h2>item detail page</h2>;
+  const { productId } = useParams();
+  const [detailProduct, setDetailProduct] = useState({});
+  const [comments, setComments] = useState([]);
+
+  const [isCommentFill, setIsCommentFill] = useState(false);
+  const [writeComment, setWriteComment] = useState('');
```

Comment:
state는 상태이므로 동사로 시작하는 것은 조금 어색해보여 다른 이름을 지어주시면 더 좋을 것 같습니다.
```suggestion
  const [userComment, setUserComment] = useState('');
```